app/models/work.rb

```diff
@@ -101,6 +101,7 @@ def dates
   def self.get_query_url(options={})
     if options[:id].present?
       params = { q: options[:id],
```

Comment:
Would this also work without the `q` parameter? There is some duplication in the current code?

Reply:
yes it does, ok; i'll remove it too

Reply:
Ah, ok.